src/WebApiToTypeScript/WebApi/WebApiController.cs

```diff
@@ -27,6 +30,7 @@ public WebApiController(TypeDefinition apiController)
             BaseEndpoint = Helpers.GetBaseEndpoint(RouteParts);
 
             BuildActions(apiController);
+            BuildMobileActions(apiController);
```

Comment:
@dsperac Do we need this? Can we do `isMobileAction: m.CustomAttributes.Any(a => a.AttributeType.Name == WebApiToTypeScript.Config.MobileEndpointAttributeName)` in the select block instead? And then when we need MobileActions, we can have a getter that filters by that property instead?